api/lib/controllers/utils.js

```diff
@@ -34,6 +34,37 @@ const propsToPrismaInclude = (props, includableFields) => {
   );
 };
 
+/**
+ * Transform props to sort into a valid prisma `sort` field
+ *
+ * @param {string} prop
+ * @param {'asc'|'dsc'} order
```

Comment:
The proper string for descending order is `desc`

```suggestion
 * @param {'asc'|'desc'} order
```